### PR TITLE
Add Negative Key To Number Pad

### PR DESCRIPTION
**Pull request by TheBrokenRail**
### Screenshot

screen

### CODE REVIEW

**Review by dekrain — APPROVED** [review]

**Comment by towerofnix** [comment]
How's this appear in the editor? A screenshot would be nice!

**Comment by dekrain** [comment]
It's probably for mobile devices

**Comment by thisandagain** [comment]
Thanks @TheBrokenRail . Could you please provide a screenshot of how this impacts the UI of the number pad?

**Comment by TheBrokenRail** [comment]
Added!

**Comment by thisandagain** [comment]
@carljbowman

File: core/field_number.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ Blockly.FieldNumber.DROPDOWN_Y_PADDING = 8;
  */
  // Calculator order
 Blockly.FieldNumber.NUMPAD_BUTTONS =
-    ['7', '8', '9', '4', '5', '6', '1', '2', '3', '.', '0', ' ', '-'];
+    ['7', '8', '9', '4', '5', '6', '1', '2', '3', '.', '0', '-', ' '];
```
Comment (jeandrek) [inline core/field_number.js:85]: Why is there a space?

**Comment by TheBrokenRail** [comment]
To Center Backspace

**Comment by mrjacobbloom** [comment]
it looks good as-is, but it could also look cool if the backspace button was 3 columns wide when on its own row

**Comment by TheBrokenRail** [comment]
Should look nicer now when negative isn't allowed

**Comment by TheBrokenRail** [comment]
Ready To Merge?

**Comment by TheBrokenRail** [comment]
?

**Comment by carljbowman** [comment]
@TheBrokenRail - Thanks for pulling this together! Looks good to me.

**Pull request merged**